types/index.d.ts

```diff
@@ -1701,6 +1701,7 @@ declare namespace Knex {
       appName?: string;
       abortTransactionOnError?: boolean;
       trustedConnection?: boolean;
+      enableArithAbort?: boolean,
```

Comment:
Could you change `,` to be `;` for consistency?

Reply:
I did.